src/app/tests/suites/include/ConstraintsChecker.h

```diff
@@ -269,6 +269,24 @@ class ConstraintsChecker
         return CheckConstraintMaxValue(itemName, current.Value(), static_cast<T>(expected));
     }
 
+    template <typename T>
+    bool CheckConstraintNotValue(const char * itemName, const chip::app::DataModel::Nullable<T> & current,
+                                 const chip::app::DataModel::Nullable<T> & expected)
+    {
+        if (expected.IsNull() && current.IsNull())
+        {
+            Exit(std::string(itemName) + " got NULL for both values, but expected not equal");
```

Comment:
```suggestion
            Exit(std::string(itemName) + " got null for both values, but expected not equal");
```
(This is not C NULL, this is Matter data model null).

Reply:
Updated. Thank you.